### ISSUE TITLE
Automatic linking with paste URL while text is highlighted?

### ISSUE THREAD

**Issue by davegaeddert** [leggett/simplify#630]
I'm curious if this could be in the scope of simplify? I'm assuming you know what I mean here... oddly hard to describe though so let me know if not.

**Comment by leggett** [comment]
Oh man, I love this interaction pattern so much. 

I currently don't listen to keypresses in compose at all but have plans to add options to do that so I can support the tab key indenting more/less. I could probably do this if I only did it on ⌘V / Ctrl+V (and not on right-click → paste).

**Comment by davegaeddert** [comment]
Just keyboard would be awesome. Honestly, I could see a potential argument for only applying it to keyboard anyway, and the right-click / edit menu become the "I want a non-fancy paste" option instead of a global setting or something to turn it off.

**Comment by leggett** [comment]
I finally got a day to work on this and am adding it to v3 in the next update. Sorry it took almost two years!

P.S. I spent an entire day building a really complicated version of this because I couldn't get the simple version working in Gmail. The complicated version took a while to build but worked perfectly the first time I tested it. And then, I immediately realized why the simple version wasn't working. I was both happy (simple is always better) and a tad sad deleting all the code for the complicated version (when does something work perfectly the first time you test it??).
